### ISSUE TITLE
import/extensions: does not complain about .coffee

### ISSUE THREAD

**Issue by ljharb** [benmosher/eslint-plugin-import#583]
I've got the following config:

``` js
['error', 'never', {
  js: 'always',
  jsx: 'always',
}]
```

However, `import '../vendor/blah.coffee';` does not warn. I would expect it to resolve the import, extract everything after the last dot, and then use that extension to determine that it should be omitted.

**Comment by benmosher** [comment]
and there is no `blah.coffee.js` file sibling, I presume?

**Comment by ljharb** [comment]
Correct.
